apps/emqx/test/emqx_bpapi_static_checks.erl

```diff
@@ -49,13 +49,17 @@
 %% List of known RPC backend modules:
 -define(RPC_MODULES, "gen_rpc, erpc, rpc, emqx_rpc").
 %% List of known functions also known to do RPC:
--define(RPC_FUNCTIONS, "emqx_cluster_rpc:multicall/3, emqx_cluster_rpc:multicall/5").
+-define(RPC_FUNCTIONS, "emqx_cluster_rpc:multicall/3, emqx_cluster_rpc:multicall/5, "
+                       "emqx_plugin_libs_rule:cluster_call/3").
```

Comment:
Q: is this from other branch? Appears to be unrelated to `emqx_resource`.

Reply:
Correct, but I packed it into the same PR, since it appears to be a minor change. It's in a separate commit, though.